pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2632,6 +2654,9 @@ func updatePortAndPciBackIoBundle(ctx *domainContext, ib *types.IoBundle) (chang
 				log.Error(err)
 			}
 		}
+		if ib.Type == types.IoNetEthPF || ib.Type == types.IoNetEthVF {
```

Comment:
I assume that we need to be able to assign the VFs to a guest domain, but is this just saying that we do not assign it to pciback/vfio-pci?

Reply:
Actually this should only apply to PFs (so that EVE can manage SRIOV functionality for the NIC) - VFs should not have `keepInHost` enabled (unless there is a reason which I missed during the design).

Reply:
It's my mistake then :D I'll fix it right away...